x/wasm/internal/keeper/proposal_integration_test.go

```diff
@@ -169,9 +176,20 @@ func TestMigrateProposal(t *testing.T) {
 	cInfo := wasmKeeper.GetContractInfo(ctx, contractAddr)
 	require.NotNil(t, cInfo)
 	assert.Equal(t, uint64(2), cInfo.CodeID)
-	assert.Equal(t, uint64(1), cInfo.PreviousCodeID)
 	assert.Equal(t, anyAddress, cInfo.Admin)
 	assert.Equal(t, "testing", cInfo.Label)
+	expHistory := []types.ContractCodeHistoryEntry{{
+		Operation: types.GenesisContractCodeHistoryType,
```

Comment:
Good, you don't store any data from the genesis one here (can be huge) - but using a different type than init shows where it came from (and where the data is if you want to query it) :+1: